components/data-table/row-actions.jsx

```diff
@@ -75,11 +84,36 @@ const DataTableRowActions = createReactClass({
 		if (isFunction(this.props.onAction)) {
 			this.props.onAction(this.props.item, selection);
 		}
+		if (this.props.dropdown && isFunction(this.props.dropdown.props.onSelect)) {
+			this.props.dropdown.props.onSelect(selection);
+		}
 	},
 
 	// ### Render
 	render () {
 		// i18n
+		const defaultDropdownProps = {
+			align: 'right',
+			buttonClassName: 'slds-button--icon-x-small',
+			buttonVariant: 'icon',
+			iconCategory: 'utility',
+			iconName: 'down',
+			iconSize: 'small',
+			iconVariant: 'border-filled',
+			assistiveText: this.props.assistiveText,
```

Comment:
do you think that we should deprecate these overlapping props, since we can have a real `Dropdown` now?  what do you think @interactivellama ?

Reply:
Yes, we should. I didn't want to require that of someone's first pull request, so I didn't say anything.